README.md

```diff
@@ -26,6 +26,9 @@ Withdraw Funds without BitGo:
   | XLM   | Yes        | Yes  |
   | XRP   | Yes        | Yes  |
   | ZEC   | Yes        | No   |
+- | HBAR  | No         | Yes  |
+- | ALGO  | No         | Yes  |
```

Comment:
```suggestion
- | HBAR  | Yes         | Yes  |
- | ALGO  | Yes        | Yes  |
```
?

Reply:
Thought to say Yes to prod once its tested by QA